### PR TITLE
Guides for migrating to Aiven

### PR DESCRIPTION
**Pull request by mortenlj**
These are the various approaches for migrating to Aiven I could think of.

### CODE REVIEW

File: addons/kafka.md [hunk 1]
```diff
@@ -150,6 +150,104 @@ Kafka requires TLS client certificates for authentication. Make sure your
...
+need to find the one that suits your situation best. Make sure you understand how the different 
+approaches affects your message processing. Some approaches might result in messages being processed 
+more than once, and some approaches might allow for messages to be processed out of order.
+
+If you are using the Schema Registry, you need to make sure that your schemas are uploaded to the 
+new schema registry in Aiven when migrating. How you do that depends on how you do your migration.
+
+The first step is always:
+
+ 0. Create the topic using the new Topic resource, and set proper ACLs
```
Comment (geiralund) [inline addons/kafka.md:166]: maybe a link to an example? (howto Topic resource?)

**Review by geiralund — APPROVED** [review]
lgtm! 
Looks like most cases are covered, cannot think of others.

File: addons/kafka.md (hunk 1 above)
Comment (mortenlj) [reply addons/kafka.md]: Fixed! 👍

**Comment by audunstrand** [comment]
lgtm

**Pull request merged**